src/main/java/bio/terra/folder/app/controller/FolderApiController.java

```diff
@@ -33,6 +39,15 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     return authenticatedUserRequestFactory.from(request);
   }
 
+  @Override
+  public ResponseEntity<JobModel> createFolder(@RequestBody CreateFolderBody createFolderBody) {
```

Comment:
not for this PR but at some point we should add input validation for folder names

Reply:
Added this in with some tests